docker-compose.yml

```diff
@@ -2,12 +2,33 @@ version: '3.3'
 
 services:
 
+  graphdb:
+    image: docker-registry.ontotext.com/graphdb-free:9.5.0-adoptopenjdk11
+    environment:
+      GDB_JAVA_OPTS: >-
+        -Dgraphdb.workbench.importDirectory=/opt/home/import-data/
+    expose:
+      - 7200
+    volumes:
+      - ./test-cypress/fixtures/graphdb-import:/opt/home/import-data/
+
   workbench:
-    container_name: graphdb-workbench
     build:
       context: .
       dockerfile: Dockerfile
     environment:
-      GRAPHDB_URL: 'http://${HOST_IP}:7200'
+      GRAPHDB_URL: 'http://graphdb:7200'
     ports:
-      - '7300:80'
+      - 80
+    depends_on:
+      - graphdb
+
+  cypress-tests:
+    build:
+      context: test-cypress
+    depends_on:
+      - workbench
+    environment:
+      - CYPRESS_baseUrl=http://workbench
+    volumes:
+      - ./coverage:/workbench/test-cypress/coverage
```

Comment:
You have to fix the permissions in the coverage directory, everything will be owned by root there, leading to "broken" workspace in Jenkins because it won't be able to delete the workspace before starting a new build.

Reply:
They are fixed in the Acceptance stage in the Jenkins file